docs/giveortake.md

```diff
@@ -17,36 +17,36 @@ Archipelago has 5 active Github repositories
   What? DevOps: Docker-compose deployment strategy, including a full skeletong project with persistance folders for Min.io, DB, Solr, Cantaloupe and Drupal 8. Includes Initial deployment configurations, what modules are enabled, how things look in Drupal 8 and some scripts plus the deployment documentation for both OSX and Linux.
 - [Archipelago Documentation](https://github.com/esmero/archipelago-documentation/tree/8.x-1.0-beta1)
   What? Documentation: This guide and whatever we manage to write to explain Archipelago goes here.
-  
+
 ## Questions, answers and in transit between both ends.
 
-For now we have a single Community interaction Channel, our [google group](https://groups.google.com/forum/#!forum/archipelago-commons). That is the best place to make questions and suggestions that are not specific to a single module and/or you think can contribute to a larger community. Discussions live better there (not excluding github of course) and our official announcements are posted there too.
+For now we have a single Community interaction Channel, our [google group](https://groups.google.com/forum/#!forum/archipelago-commons). That is the best place to ask questions and make suggestions that are not specific to a single module and/or you think can contribute to a larger community. Discussions work best in this forum (not excluding github of course), and our official announcements are posted there too.
 
 
 ## Documentation Workflow
 
-Documentation is right now an evolving effort and we need help. This guide lives in github in [Archipelago Documentation](https://github.com/esmero/archipelago-documentation/tree/8.x-1.0-beta1). Documentation and Development Worklfow behaves in the same way, so keep reading!
+Documentation is an evolving effort, and we need help. This guide lives in github in [Archipelago Documentation](https://github.com/esmero/archipelago-documentation/tree/8.x-1.0-beta1). Documentation and Development Worklfow both work the same way, so keep reading!
 
 ## Development Workflow
 
-Start by reading open ISSUES (so you don't end redoing what someone else is alrady working on) and looking at our [Roadmap for version 8.x-1.0](https://github.com/esmero/archipelago-deployment/issues/5). If your need is not there or there is an unchecked element in the roadmap you can help with.
+Start by reading open ISSUES (so you don't end redoing what someone else is already working on) and looking at our [Roadmap for version 8.x-1.0](https://github.com/esmero/archipelago-deployment/issues/5). If the solution to your problem is not there or of there is an unchecked element in the roadmap, this is a great opportunity to help by creating a new ISSUE.
```

Comment:
I think that in  `is not there or of there`, the  **of**  is off. But i can be wrong

Reply:
I'll get it in my next pass :)